components/vehiclespawner/fn_canLogiSpawnVehicle.sqf

```diff
@@ -0,0 +1,26 @@
+/*
+	Check if logi vehicle is currently capable of spawning the object stored at the provided logi object index.
+	The logi vehicle must have been initialised, must not currently be spawning anything and it must currently not have run out of stock of the object.
+
+	Author: Bubbus
+
+	Passable arguments:
+	0: Logi vehicle to test capability upon
+	1: The logi object index to test.  This is ordered identically to configuration.
+
+	Returns:
+	true iff logi vehicle is capable of spawning an object at the given object index at the current time.
```

Comment:
when IFF is so important it becomes part of the doc

***Alpha B-1-2:** "Unknown argument, 200 meters, front!"*

(sorry)

Reply:
The only part of 1st year comp sci i remembered lol

https://en.wikipedia.org/wiki/If_and_only_if